module/VuFind/src/VuFind/Search/Factory/SolrDefaultBackendFactory.php

```diff
@@ -43,6 +43,13 @@
  */
 class SolrDefaultBackendFactory extends AbstractSolrBackendFactory
 {
+    /**
+     * Callback for creating a record driver.
+     *
+     * @var string
+     */
+    protected $createRecordCallback = 'getSolrRecord';
```

Comment:
It might be better to call this `$createRecordMethod` since it's not a full PHP callback, just a method name for the plugin manager.

Reply:
I agree, renamed.

Reply:
Thanks!